src/components/FormEditor.tsx

```diff
@@ -378,7 +385,7 @@ export const FormEditor = React.forwardRef<HTMLDivElement, FormEditorProps>(
                         endLineNumber: p.lineNumber,
                         endColumn: p.column,
                     },
-                    text: files.map((url) => mdImageLink(url)).join(', '),
+                    text: files.map((file) => mdImageLink(file)).join(', '),
```

Comment:
Really needs join values with comma in this case? Probably for images it must be a `new line` character